assertj-core/src/test/java/org/assertj/core/api/localdatetime/LocalDateTimeAssert_isEqualTo_Test.java

```diff
@@ -16,80 +16,31 @@
 package org.assertj.core.api.localdatetime;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.mockito.Mockito.verify;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 
 import java.time.LocalDateTime;
-import java.time.chrono.ChronoLocalDateTime;
-import java.time.chrono.JapaneseChronology;
-import java.time.chrono.JapaneseDate;
-import java.time.format.DateTimeParseException;
 
-import org.assertj.core.api.AbstractLocalDateTimeAssertBaseTest;
-import org.assertj.core.api.LocalDateTimeAssert;
-import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
 import org.junit.jupiter.api.Test;
 
 /**
- * Only test String based assertion (tests with {@link java.time.LocalDateTime} are already defined in assertj-core)
- *
- * @author Joel Costigliola
- * @author Marcin Zajączkowski
+ * Tests for {@link org.assertj.core.api.LocalDateTimeAssert#isEqualTo(String)} with invalid date strings.
+ * 
+ * @author Your Name
  */
-class LocalDateTimeAssert_isEqualTo_Test extends AbstractLocalDateTimeAssertBaseTest {
```

Comment:
keep the existing tests except `should_fail_if_given_localDateTime_as_string_parameter_cant_be_parsed` of course